dev/src/components/__tests__/CourseCreationInfo.test.tsx

```diff
@@ -0,0 +1,20 @@
+import CourseCreationInfo from "../admin/CourseCreationInfo";
+
+import {configure, shallow} from 'enzyme'
+import Adapter from "@wojtekmaj/enzyme-adapter-react-17"
+import React from "react";
```

Comment:
Inconsistent use of `;`. I have installed the ESLint plugin in VSCode and enabled formatting with this:
```json
    "editor.codeActionsOnSave": {
        "source.fixAll": true
    },
```
If you do this, it should fix these things on save.